source/includes/quick-start/query-output.rst

```diff
@@ -0,0 +1,15 @@
+When you run ``main.go``, it should output the details of the
```

Comment:
If we are assuming users are relatively new to Go, I suggest adding an explicit step for running `main.go` on the CLI:

```
go run main.go
```

Similar to the Node docs.